languages/move_on_aptos/generic/Parse_move_on_aptos_tree_sitter.ml

```diff
@@ -8,6 +8,10 @@ module H2 = AST_generic_helpers
 (*****************************************************************************)
 (* Helpers *)
 (*****************************************************************************)
+
+(* Disable warnings against unused variables *)
+[@@@warning "-26-27-32"]
+
```

Comment:
I think it makes sense to colocate this with the other directive to disable warnings. Could you either move this back or move the other one up here too?

Reply:
I can only move the another one up because some of the helper functions were triggering the warnings.

Reply:
Also it would actually be better to fix those warnings.
If you know it's ok for a variable is unused, you can prefix with an underscore, as in let _never_used = ...
and that will silence too the warning.